.env.example

```diff
@@ -77,4 +77,8 @@ OPENNODE_KEY=OPENNODE_ADMIN_KEY
 
 # FakeWallet
 FAKE_WALLET_SECRET="ToTheMoon1"
-LNBITS_DENOMINATION=sats
\ No newline at end of file
+LNBITS_DENOMINATION=sats
+
+# Strike
+STRIKE_API_ENDPOINT="https://api.strike.me/"
+STRIKE_API_KEY="my api key"
```

Comment:
It should be in the extension

Reply:
I'm still thinking about how to do it in the best way. 

I don't like the idea of putting it into the extensions, because it brings an unnecessary burden to the consumer. 

Additionaly it raises security questions, because we can't collect API keys in plaintext, we would have to encrypt values in the db somehow (so even the developer/admin would not have access to it). 

But it feels like reinventing the wheel, because we have OAuth standard in place that can handle it elegantly without storing any user secret data ([example here](https://www.reacher.me/)). 
Unfortunately, this auth method is not yet available for general use.

Reply:
Having the `STRIKE_API_KEY` and `STRIKE_API_ENDPOINT` in the .env file is going to be problematic for, as one example, Umbrel users. They're not expected to have to modify .env files to use LNBits today. Adding these vars will require them to if they want to use the Strike extension.

Also, adding these env vars will place a new burden on LNBits hosts (like @arcbtc and myself). We'd have to create API keys tied to our own accounts (what if we're not US/ES/AR based?) and let other people use them. At any time, someone could break a rule and get those keys shut down by Strike.

Obviously, storing API keys as plaintext in a database that a host admin can read isn't great. But, after taking a look at the [Strike API Reference](https://docs.strike.me/api/get-events), it doesn't seem like API keys allow a way to send out funds. This definitely puts a blast radius limit on storing API Keys in plaintext in the db. What could a developer gain from user's API Keys? They can't be used to get money, check a user's balance, etc.

I'm assuming the big concern here is that Strike may update the API in the future to enable sending funds, checking balances, etc. And then LNBits hosts would be sitting on a hot potato. But, if they do, surely they'd add new permissions to the keys that would be disabled by default. 

I don't think this is too big a burden on users to set up their own Strike API Key for their own account. LNBits requires users to do the same today in the Spotify Jukebox extension, for example.

So, yeah, rock and a hard place. Best practice vs. practical implementation with what's available now.

Just throwing some thoughts out there in hopes of re-kindling this conversation. I'm not married to any option.

Reply:
I see, yea, my concern is that after Strike enables sending funds, users won't pay too much attention to the scopes (restriction) when creating new API Keys. So it could potentially jeopardize Strike in case of API keys leak. 

But ok, let's do it this way, I think it is only a temporary solution as we can adopt proper OAuth flow with consent screen in the future, without any API keys.

But can we leave both options for now (DB and .env file)? DB config will override .env one. 
Strike will likely be paying some reward to the middle-mans that are issuing LN invoices on behalf of other users. So it makes sense that the admin/operator of the Lnbits will be motivated to configure their own API key to collect a rebate.

Please push your commit, you should have writing access.

Reply:
I like that solution (allowing both options for now).  Pushed.